src/server/auth/server/api_server.go

```diff
@@ -436,11 +313,16 @@ func (a *apiServer) Activate(ctx context.Context, req *auth.ActivateRequest) (re
 	}
 
 	// Store a new Pachyderm token (as the caller is authenticating) and
-	// initialize admins (watchAdmins() above will see the write)
+	// initialize the root user as a cluster admin
 	if _, err = col.NewSTM(ctx, a.env.GetEtcdClient(), func(stm col.STM) error {
-		admins := a.admins.ReadWrite(stm)
+		roleBindings := a.roleBindings.ReadWrite(stm)
 		tokens := a.tokens.ReadWrite(stm)
-		if err := admins.Put(auth.RootUser, epsilon); err != nil {
+		if err := roleBindings.Put(clusterRoleBindingKey, &auth.RoleBinding{
+			Entries: map[string]*auth.Roles{
+				auth.RootUser: &auth.Roles{Roles: map[string]bool{auth.ClusterAdminRole: true}},
+				auth.PpsUser:  &auth.Roles{Roles: map[string]bool{auth.ClusterAdminRole: true}},
```

Comment:
Explicitly include the PPS user in the cluster role bindings. These bindings can't be removed, but they make it clearer how the "PPS superuser" code works IMO.